06_create_cluster.sh

```diff
@@ -36,4 +36,20 @@ if [[ ! -z "${ENABLE_LOCAL_REGISTRY}" ]]; then
       --type merge --patch "{\"spec\":{\"samplesRegistry\":\"${LOCAL_REGISTRY_DNS_NAME}\"}}"
 fi
 
+if [[ ! -z "${APPLY_EXTRA_WORKERS}" ]]; then
```

Comment:
let's use -n instead ?

Reply:
It seems that this usage is more common in dev-scripts

Reply:
that's true, and IMHO that's unfortunate
Here (and in other places in dev-scripts) we want to check if the variable is set and not empty, the canonical way to do that in Bourne-like shell is using -n, or you could use -v in modern versions (>=4.2)
the rest is just a complication